redisvl/schema/schema.py

```diff
@@ -144,7 +144,10 @@ class IndexSchema(BaseModel):
     index: IndexInfo
     """Details of the basic index configurations."""
     fields: Dict[str, BaseField] = {}
-    """Fields associated with the search index and their properties"""
+    """Fields associated with the search index and their properties.
+
+    Note: When creating from dict/YAML, provide fields as a list of field definitions.
```

Comment:
Does it support both input types now even if it normalizes to dict?

Reply:
Done